### DISCUSSION TITLE
My Github page is not loading well after connecting to a custom domain

### DISCUSSION THREAD

**Discussion by annamisak** [community/community#73398] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi! 
I am not sure if this question is being posted in the correct area, but I hope it is :)

I am currently having some issues. I connected my Github page to a custom domain. The connection seems to be good but for some reason it does not connect the page correctly. All the CSS styles are not loading.

Here is the [link to my page](https://annamisak.github.io/annamisak.com/).

I wonder what I did wrong, I would appreciate advice on how to fix this issue.

**Comment by subidit** [comment, marked as answer]
You should rename your repository to 
`<username>.github.io`.

Rest all seems ok.

**Reply by annamisak** [reply]
Thank you so much for your advice, this seemed to fix the problem. The page is loading good now :)
